src/ngraph/runtime/nvgpu/cublas_emitter.hpp

```diff
@@ -19,20 +19,20 @@
 #include <cublas_v2.h>
 
 #include "ngraph/op/dot.hpp"
-#include "ngraph/runtime/gpu/gpu_runtime_context.hpp"
+#include "ngraph/runtime/nvgpu/nvgpu_runtime_context.hpp"
 #include "ngraph/shape.hpp"
 
 namespace ngraph
 {
     namespace runtime
     {
-        namespace gpu
+        namespace nvgpu
         {
-            class GPUPrimitiveEmitter;
+            class NVPrimitiveEmitter;
```

Comment:
Nit: My personal feeling is `NVGPUxxx` or just `xxx` in a `nvgpu` namespace instead of `NVxxx`. `NV` is not intuitive in my mind. This apply to all other variable or class name.